app/konnect-platform/cmek.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Customer-Managed Encryption Keys (CMEK)
+content_type: reference
+layout: reference
+description: 'Use Customer-Managed Encryption Keys (CMEK) in {{site.konnect_short_name}} to encrypt sensitive data using keys from your AWS Key Management Service (KMS) account.'
```

Comment:
Should we say "to encrypt pre-determined set of sensitive data"? I want to highlight that cmek can only support a  subset of data.